change/@fluentui-react-portal-34061c16-8654-47b2-8a3e-739f4f8e0ac9.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
```

```suggestion
  "type": "patch",
```

Should not be this patch?